src/files-and-videos/videos/info-sidebar/TranscriptTab.test.jsx

```diff
@@ -0,0 +1,239 @@
+import {
+  render,
+  act,
+  fireEvent,
+  screen,
+  waitFor,
+} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import ReactDOM from 'react-dom';
+
+import {
+  initializeMockApp,
+} from '@edx/frontend-platform';
+import MockAdapter from 'axios-mock-adapter';
+import { getAuthenticatedHttpClient } from '@edx/frontend-platform/auth';
+import { AppProvider } from '@edx/frontend-platform/react';
+import { IntlProvider } from '@edx/frontend-platform/i18n';
+
+import initializeStore from '../../../store';
+import { executeThunk } from '../../../utils';
+import { RequestStatus } from '../../../data/constants';
+import TranscriptTab from './TranscriptTab';
+import {
+  courseId,
+  initialState,
+} from '../factories/mockApiResponses';
+
+import { getApiBaseUrl } from '../data/api';
+import messages from './messages';
+import transcriptRowMessages from './transcript-item/messages';
+import VideosProvider from '../VideosProvider';
+import { deleteVideoTranscript } from '../data/thunks';
+
+ReactDOM.createPortal = jest.fn(node => node);
```

Comment:
Where are we using portal? Did I overlook it somewhere in the PR?

Reply:
Not using portal. An accidental import when I was copying and pasting.